batcherror/batch.go

```diff
@@ -89,22 +89,28 @@ func (be *BatchError) addBatch(batch BatchError) {
 	be.errors = append(be.errors, batch.errors...)
 }
 
-// Add adds an error into the batch.
+// Add adds errors into the batch.
 //
-// If the error is also an BatchError,
+// If an error is also an BatchError,
 // its underlying error(s) will be added instead of the BatchError itself.
 //
-// Nil error will be skipped.
-func (be *BatchError) Add(err error) {
-	if err == nil {
+// Nil errors will be skipped.
+func (be *BatchError) Add(errs ...error) {
+	if errs == nil {
 		return
 	}
```

Comment:
nit: This if is no longer needed (in that case the for loop will just be executed 0 times).